src/langcheck/augment/_common/_jailbreak_template.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+import random
+
+from langcheck.metrics.prompts._utils import get_template
+
+
+def _jailbreak_template(
+    instances: list[str] | str,
+    templates: list[str] | None,
+    available_templates: list[str],
+    language: str,
+    *,
+    num_perturbations: int = 1,
```

Comment:
maybe allow setting a seed to make the selected templates deterministic?

Reply:
Added.

Reply:
Thanks!

Maybe we can also allow this param to be `None`, which would mean that all of the perturbations will be iteratively run in the provided order? In addition to _knowing_ the order of the templates that were used (i.e. comment below), it'll be nice to be able to _control_ the order. (Randomizing the order of applied templates is probably not that useful when `num_perturbations == len(templates)` anyway)

Reply:
Related to the other reply, but since all the other augment methods use `num_perturbations: int` parameter so I added the argument with the same type. I think it is better to keep that. I also think it would be more natural to add another parameter like `use_all_templates: bool` for your use case. WDYT?

Reply:
Ohh I see, didn't realize we had `num_perturbations` for other functions.

Adding another param seems ok too, but we'd need to think of a good name that conveys the notion of "turn off order randomization when num_perturbations == len(templates)". How about `randomize_order: bool`, and then throw a ValueError when it's False but `num_perturbations < len(templates)`?

Reply:
Sounds good, added the option.